index.html

```diff
@@ -6,13 +6,178 @@
+                    <span class="hamburger-top"></span>
+                    <span class="hamburger-middle"></span>
+                    <span class="hamburger-bottom"></span>
+                </button>
+            </div>
+            <div class="logo pr-12 md:pl-24">
+                <a href="index.html" class="text-2xl font-bold">Hum Style Awards</a>
+            </div>
+            <div class="menu-links hidden md:flex md:items-center  space-x-6 pr-6 ">
+                <a href="about.html" class="text-lg" rel="noopener">About</a>
+                <a href="#program" class="text-lg" rel="noopener">Program</a>
+                <a href="#speakers" class="text-lg" rel="noopener">Speakers</a>
+                <a href="#partners" class="text-lg py-2 px-4 campaign" rel="noopener">CC Campaign</a>
+            </div>
+        </nav>
+        <ul id="menu"
+            class="absolute top-0 left-0 hidden flex-col self-end w-full min-h-screen py-1 pt-40 pl-12 space-y-3 text-lg text-white uppercase overlay ">
+            <li class="mobile-links"> <a href="index.html" class="text-lg" rel="noopener">Home</a></li>
+            <li class="mobile-links"><a href="about.html" class="text-lg" rel="noopener">About</a></li>
+            <li class="mobile-links"> <a href="#program" class="text-lg">Program</a></li>
+            <li class="mobile-links"> <a href="#speakers" class="text-lg">Speakers</a></li>
+            <li class="mobile-links"> <a href="#partners" class="text-lg py-2 px-4 campaign self-start">CC
+                    Campaign</a></li>
+        </ul>
+    </header>
+    <main>
+        <!-- hero section -->
+        <section class="hero w-full py-20" id="hero">
+            <div class="hero-content md:pt-6  pb-12 md:pb-12 px-8 md:px-36 flex items-center flex-col md:block">
+                <div>
+                    <h2 class="hero-light text-3xl md:text-4xl">"Hello Good people!"</h2>
+                    <h1 class="hero-title font-black text-5xl md:text-7xl">Hum Style Awards <br> 2022</h1>
+                </div>
+                <div class="hero-desc max-w-lg mt-4 md:max-w-3xl text-sm md:text-base  py-2 px-4">
+                    <p>
+                        The Hum Awards are one of the most prestigious awards in the Pakistan entertainment industry,
+                        and
+                        this year’s nominations are no exception. From Best Female Actress Popular to Best Drama Serial
+                        Popular, there are plenty of deserving nominees. The 8th Hum Awards Nominations for 2022 are
+                        listed
+                        below.
+                    </p>
+                </div>
+                <h2 class="text-3xl mt-6 md:mt-4 md:text-5xl text-center md:text-start hero-date font-black">2022-09-24
+                    (Sat)
+                </h2>
+                <h3 class="text-center md:text-start text-2xl mt-6 md:mt-4 md:text-2xl hero-venue">@ FirstOntario Centre
```

Comment:
- [OPTIONAL] Kindly take note of the suggestion. in the image below

| Your Design |
|-------------|
| ![image](https://user-images.githubusercontent.com/62938735/190687188-6899e594-94a5-4aee-9668-b80a4cad33a2.png)|